views/js/runner/provider/qti.js

```diff
@@ -656,7 +656,7 @@ define([
         unloadItem : function unloadItem(){
             var self = this;
 
-            self.trigger('disablenav disabletools');
+            self.trigger('beforeunloaditem disablenav disabletools');
```

Comment:
That was my first plan but actually, due to the test runner architecture, the event is triggered once the action has been delegated and performed by the provider. So, using before would still launch the callback once the action is performed...